sqladmin/application.py

```diff
@@ -34,13 +41,26 @@ def __init__(
         base_url: str = "/admin",
         title: str = "Admin",
         logo_url: str = None,
+        language: str = None,
     ) -> None:
         self.app = app
         self.engine = engine
         self.base_url = base_url
         self._model_admins: List[Type["ModelAdmin"]] = []
 
         self.templates = Jinja2Templates("templates")
+        self.templates.env.add_extension("jinja2.ext.i18n")
+        if language:
+            translation = gettext.translation(
```

Comment:
I'd rather do this in a separate changes to keep small PR and make review easier. What do you think?

Reply:
I'll pay attention next time.

If you think I submit too many contents this time, I can submit them separately.

Reply:
that would be great. If you could focus this PR on the login functionality and do other PRs for translation and other things I'd really appreciate that.
PRs are always appreciated :)